core/src/Kokkos_Complex.hpp

```diff
@@ -798,25 +800,25 @@ template <class RealType>
 KOKKOS_INLINE_FUNCTION Kokkos::complex<RealType> sqrt(
     const complex<RealType>& x) {
 #ifdef KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_SYCL
-  using sycl::abs;
+  using sycl::fabs;
   using sycl::sqrt;
 #else
-  using std::abs;
+  using std::fabs;
   using std::sqrt;
 #endif
 
   RealType r = x.real();
   RealType i = x.imag();
 
   if (r == RealType()) {
-    RealType t = sqrt(abs(i) / 2);
+    RealType t = sqrt(fabs(i) / 2);
     return Kokkos::complex<RealType>(t, i < RealType() ? -t : t);
   } else {
-    RealType t = sqrt(2 * (abs(x) + abs(r)));
+    RealType t = sqrt(2 * (abs(x) + fabs(r)));
```

Comment:
```suggestion
    RealType t = sqrt(2 * (fabs(x) + fabs(r)));
```
?

Reply:
No.  I did the same mistake before. `x` is a `Kokkos::complex`.